tests/test_misc/server_test_unix.rs

```diff
@@ -0,0 +1,66 @@
+#![allow(dead_code)]
+
+use std::sync::Arc;
+
+use futures::stream;
+
+use bytes::Bytes;
+
+use httpbis::Headers;
+use httpbis::HttpPartStream;
+use httpbis::Response;
+use httpbis::Server;
+use httpbis::ServerBuilder;
+use httpbis::Service;
+
+use regex::Regex;
+
+/// HTTP/2 server used by tests
+pub struct ServerTestUnixSocket {
+    pub server: Server<String>,
+}
+
+
+struct Blocks {}
+
+impl Service for Blocks {
```

Comment:
Yes, I copy-pasted the server_test.rs file, then made small changes to make a UDS ServerTest struct..
I will refactor them back together and clean up a little bit.